### PR TITLE
YCDOCIO-2104: Add YDS to OBJ Storage migration

### PR DESCRIPTION
**Pull request by voloschuk-fedor**
https://st.yandex-team.ru/YCDOCIO-2104

### CODE REVIEW

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Infrastructure for the Yandex Cloud Data Streams, Object Storage and Data Transfer.
```
Comment (redalka-cloud) [inline tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:1]: Infrastructure for the Yandex Cloud Data Streams, Object Storage, and Data Transfer.

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  bucket_name = "" # Set a Object Storage bucket name. It must be unique throughout Object Storage.
+  #target_endpoint_id = "" # Set the target endpoint id.
+}
+
+resource "yandex_iam_service_account" "sa-yds-obj" {
+  description = "Service account for migration from the Data Streams to Object Storage"
+  name        = local.sa_name
+}
+
+# Assign role `yds.editor` to the service account.
```
Comment (redalka-cloud) [inline tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:25]: Assign the `yds.editor` role to the service account.

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+locals {
+  folder_id = "" # Your Folder ID.
+  sa_name   = "" # Set a service account name. It must be unique in a cloud.
+
+  # Source database settings:
+  source_db_name = "" # Set the source database name.
+  #source_endpoint_id = "" # Set the source endpoint id.
+
+  # Target bucket settings:
+  bucket_name = "" # Set a Object Storage bucket name. It must be unique throughout Object Storage.
```
Comment (redalka-cloud) [inline tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:16]: Set an Object Storage bucket name

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf (hunk 1 above)
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:1]: Ок

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf (hunk 2 above)
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:25]: Ок

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+# Assign role `yds.editor` to the service account.
+resource "yandex_resourcemanager_folder_iam_binding" "yds_editor" {
+  folder_id = local.folder_id
+  role      = "yds.editor"
+  members = [
+    "serviceAccount:${yandex_iam_service_account.sa-yds-obj.id}",
+  ]
+}
+
+# Assign role `storage.editor` to the service account.
```
[the remark this answers is not in the document]
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:34]: Ок

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+resource "yandex_resourcemanager_folder_iam_binding" "storage_editor" {
+  folder_id = local.folder_id
+  role      = "storage.editor"
+  members = [
+    "serviceAccount:${yandex_iam_service_account.sa-yds-obj.id}",
+  ]
+}
+
+# Assign role `storage.uploader` to the service account.
```
[the remark this answers is not in the document]
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-yds-obj.tf:43]: Ок

File: tutorials/terraform/data-transfer/data-transfer-yds-obj.tf (hunk 3 above)
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-yds-obj.tf]: Ок

**Pull request merged**